tests/integration/test_case.go

```diff
@@ -233,6 +233,9 @@ type CreateDoc struct {
 	// Specifies whether the document should be encrypted.
 	IsEncrypted bool
 
+	// Individual fields of the document to encrypt.
+	EncryptedFields []string
```

Comment:
todo: Please add tests where non-existing fields are added to this set.  If you are able to make the GQL input type a `[Enum]` you will unfortunately need to split the tests by mutation types as the errors should be different (unless the GQL error is suitable for re-use via the collection API).

Reply:
done